spec/support/matchers.rb

```diff
@@ -94,3 +94,24 @@
     "expected that #{actual.url} would link to valid URLs, but found:\n#{bad_urls.join("\n")}"
   end
 end
+
+RSpec::Matchers.define :be_uniquely_titled do
+  # Attempts to validate conformance to WCAG Success Criteria 2.4.2: Page Titled
+  #
+  # Visiting a page with the default app name title is considered a failure, and should be resolved
+  # by providing a distinct description for the page using the `:title` content block.
+  #
+  # https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html
+
+  title = nil
+
+  match do |doc|
+    title = doc.css('title').first.text.strip
+
+    !title.empty? && title.strip != '| login.gov'
```

Comment:
Any reasonably straight-forward way we could read this from the actual site config, to match how it's rendered in the partial using `site.title` ?

Reply:
Technically, we could parse the YAML frontmatter of the source, then we have to load i18n to look up the values

However I think this is the best way to capture the actual rendered value? Since there's some shenanigans in `meta.html`, especially for help page titles, would rather not risk any re-implementation not matching the rendered result

Reply:
But we don't care about the specific page title, so isn't it enough to parse `_config.yml` to get the `title` property?  

e.g. 

```suggestion
    !title.empty? && title.strip != "| #{YAML.load_file('../../_config.yml')['title']}"
```

Reply:
Oh!!! I see now!! I misunderstood. Yes, for the `login.gov` part that seems super reasonable. I'll give that a shot

Reply:
Updated in 2c8c03232a24ee69f40abfdd97e7f8696be0b895